openbas-api/src/main/java/io/openbas/rest/injector_contract/InjectorContractApi.java

```diff
@@ -89,8 +71,7 @@ public Page<InjectorContract> injectorContracts(@RequestBody @Valid final Search
             }
         }
         return buildPaginationJPA(
-                (Specification<InjectorContract> specification, Pageable pageable) -> this.injectorContractRepository.findAll(
-                        specification, pageable),
+                this.injectorContractRepository::findAll,
```

Comment:
Why are we not using the Specification anymore ? Wouldn't that impact the sorting and paging ?

Reply:
This is a method reference expression. We are still using specification and pageable